sdk/keyvault/azure-keyvault-keys/pyproject.toml

```diff
@@ -39,7 +48,12 @@ version = {attr = "azure.keyvault.keys._version.VERSION"}
 readme = {file = ["README.md"], content-type = "text/markdown"}
 
 [tool.setuptools.packages.find]
-exclude = ["samples*", "tests*", "azure", "azure.keyvault"]
+exclude = [
```

Comment:
Samples has been removed from here as per discussion: https://github.com/microsoft/typespec/issues/7714

Reply:
@lmazuel - We don't check this field in codegen rn. Do we want to also want to keep any extra excluded items here?